src/main/resources/hazelcast-config-schema.json

```diff
@@ -2051,6 +2051,49 @@
           "description": "Specifies the directory where the non-volatile memory (e.g. Intel Optane) is mounted. \\n If this element is not defined, the RAM is used as a native memory. \\n This directory will be created automatically if it does not exist.\\n To avoid collisions, every member of the cluster will create its own subfolder to work with the non-volatile memory."
         }
       }
+    },
+    "CrdtReplication": {
+      "description": "Configures the replication mechanism for all CRDT implementations.\\n The CRDT states are replicated in rounds (the period is configurable) and \\n in each round the state is replicated up to the configured number of members",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "max-concurrent-replication-targets": {
+          "description": "The maximum number of target members that we replicate the CRDT states to in one period.",
+          "type": "integer",
+          "minimum": 1
+        },
+        "replication-period-millis": {
+          "description": "The period between two replications of CRDT states in milliseconds.",
+          "type": "integer",
+          "minLength": 1,
```

Comment:
I think it can be deleted (`"minLength"` is only applicable to string instances)